network/netcore/src/transport/tcp.rs

```diff
@@ -75,27 +75,27 @@ impl Transport for TcpTransport {
     ) -> Result<(Self::Listener, NetworkAddress), Self::Error> {
         let ((ipaddr, port), addr_suffix) =
             parse_ip_tcp(addr.as_slice()).ok_or_else(|| invalid_addr_error(&addr))?;
```

Comment:
move parse_ip_tcp into network_address.rs / NetworkAddress?

Reply:
it already is : p

Reply:
Oooh, that's interesting... checking apis...
oh two questions about that
1) Why aren't these implemented directly on NetworkAddress
2) Why not import the module and call the functions that way, at least old farts like me wouldn't make such bold and inaccurate statements :)